docs/token-format.md

```diff
@@ -0,0 +1,48 @@
+
+- `grant_type` (always set to `urn:ietf:params:oauth:grant-type:token-exchange`
+- `subject_token`
+- `subject_token_type`
+
+Issued access tokens can be restricted further through the use of `audience`, `resource`, and `scope` parameters in the token request. `audience` and `resource` restrict the class of resources for which the issued token may be used. `scope` restricts the type of actions the token can perform on resources.
+
+For the first iteration of identity-manager-sts, we are only supporting the use of `resource`, which will allow us to expose a set of services that a user may use to restrict the context in which the token is valid. The `resource` parameter maps to the `aud` claim on the issued token, as outlined in [RFC 9068][rfc-9068].
+
+
+### Token Response
+
+The token returned is an [RFC 9068][rfc-9068]-compliant access token JWT with the following claims:
+
+| claim              | description                                                                                         |
+|--------------------|-----------------------------------------------------------------------------------------------------|
+| iss                | FQDN for identity-manager-sts issuer                                                                |
+| iat                | The time the token was issued                                                                       |
+| jti                | Unique ID for the token issued                                                                      |
+| exp                | Token expiration time                                                                               |
+| sub                | Subject of the token in the infratographer namespace<br>example "sub": `infratographer:user:{uuid}` |
+| aud                | Resources on which that token may operate                                                           |
+| client_id          | ID of the client requesting the token                                                               |
+| infratographer.sub | Private claim which indicates the subject be used in policy enforcement                             |
+
+The following are defined in [RFC 8693][rfc-8693], but for now aren't supported until we run into/identify the use cases for them:
+
+- `act` describes the acting party to whom access has been delegated. identity-manager-sts currently only supports impersonation semantics, not delegation.
+- `may_act` claims in the subject_token are used together with `actor_token` to determine if the actor is authorized to proceed as the authority for the original subject.
+
+
+### Claim Mapping
+
+In many scenarios, organizations may have their own set of ways of assigning permissions to users, and may want that to map the same way when operating with an exchanged token. In order to accomplish this, a user may provide custom claim mappings when defining an issuer in identity-manager-sts.
```

Comment:
Do we have documentation for the claim mappings mechanisms? We probably want to link to those as this docs assumes the reader has that knowledge.

Reply:
We do in the README, but we should add a document around what that means to this repo. Should that go before, after, or with this PR?

Reply:
let's add that in another PR.